telegram/base.py

```diff
@@ -74,9 +74,17 @@ def to_dict(self):
 
             value = self.__dict__[key]
             if value is not None:
-                if hasattr(value, 'to_dict'):
-                    data[key] = value.to_dict()
+                if type(value) == list:
```

Comment:
Update from a private chat: I was wrongfully thinking of `de_json` instead of `to_dict`, but the outcome is the same.